# 6.0.6: path is mozilla/directory/c-sdk/configure.in
# 6.0.7: path is changed to c-sdk/configure.in

--- c-sdk/configure.in	Tue Feb 16 10:21:41 2010
+++ c-sdk/configure.in	Tue Feb 16 10:28:27 2010
@@ -5511,13 +5511,8 @@
     	DSO_LDOPTS='-G -h $(notdir $@)'
         DSO_CFLAGS=-KPIC
     	if test -n "$USE_64"; then
-    		if test "$OS_TEST" = "i86pc"; then
-    			CC="$CC -xarch=amd64"
-            	CXX="$CXX -xarch=amd64"
-            else
-            	CC="$CC -xarch=v9"
-            	CXX="$CXX -xarch=v9"
-            fi
+		CC="$CC -m64"
+            	CXX="$CXX -m64"
         fi
         CC="$CC -xstrconst"
         if test -z "$MOZ_OPTIMIZE"; then
